x/application/client/cli/helpers_test.go

```diff
@@ -0,0 +1,44 @@
+package cli_test
```

Comment:
Is there a reason not to promote this to a shared helper? I would expect that there will be scenarios from tests in other packages which will need to the same or  similar things. Perhaps we should make an appropriate package for this and put it in either the `testutil` or a  new `internal` package in the repo root.